spec/requests/api/v1/registrations_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Registrations', type: :request do
+
+  let(:valid_attributes) do
+    { email: 'test@test.com', gender: 'male', password: '123456', password_confirmation: '123456' }
+  end
+
+  let(:invalid_attributes) do
+    { email: 'test@test.com', gender: 'male_bat', password: '123456', password_confirmation: '123456' }
+  end
+
+  describe 'POST /create' do
+    context 'with valid parameters' do
```

Comment:
Nice. Add another test that validates the user information receveived is actually the data store in the database.